acceptance/lib/pxp-agent/test_helper.rb

```diff
@@ -88,11 +88,23 @@ def run_pcp_broker(host, instance=0)
 
 def kill_pcp_broker(host)
   on(host, "ps -C java -f | grep pcp-broker | sed 's/[^0-9]*//' | cut -d\\  -f1") do |result|
-    pid = result.stdout.chomp
-    on(host, "kill -9 #{pid}")
+    pids = result.stdout.chomp.split("\n")
+    pids.each do |pid|
+      on(host, "kill -9 #{pid}")
+    end
   end
 end
 
+def block_pcp_broker(host, instance)
+  host[:pcp_broker_instance] = instance+1
```

Comment:
What's the point of line 99? If host[:pcp_broker_instance] stores the broker index that should be considered the current broker, is it correct for this method to change it? We might have HA tests where we want to block a broker that is neither the current nor the next broker we want to use.

And is the +1 because we're incrementing, or because we're moving from 0-index to 1-index? If it's the former, then does this method work when we're on the last broker and need to cycle back round? If it's the latter, then is line 104 correct?